content/blog/2022/01/2022-01-06-gsoc-2022.adoc

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "Google Summer of Code 2022 call for Project Ideas and Mentors"
+tags:
+- gsoc
+- gsoc2022
+- community
+- events
+- developer
+authors:
+- alyssat
+- jmMeessen
+description: >
+  This blogpost calls for projects and mentors for the 2022 edition of Google Summer of Code.
+links:
+  discourse: true
+---
+
+link:https://summerofcode.withgoogle.com[Google Summer of Code] (GSoC) is a global, online mentoring program focused on introducing new contributors to open source software development. 
+GSoC contributors work on a 12+ week programming project with the guidance of mentors from their open source organization.  
+During GSoC, participating contributors are paired with mentors from open source organizations, gaining exposure to real-world software development techniques. 
+GSoC contributors will learn from experienced open source developers while writing code for real-world projects! 
```

Comment:
Best to remove trailing white space from the end of lines.  Not blocking merge, but a preference
```suggestion
link:https://summerofcode.withgoogle.com[Google Summer of Code] (GSoC) is a global, online mentoring program focused on introducing new contributors to open source software development.
GSoC contributors work on a 12+ week programming project with the guidance of mentors from their open source organization.
During GSoC, participating contributors are paired with mentors from open source organizations, gaining exposure to real-world software development techniques.
GSoC contributors will learn from experienced open source developers while writing code for real-world projects!
```